scripts/lib.sh

```diff
@@ -36,12 +36,16 @@ get_from_kata_deps() {
 	local branch="${2:-${BRANCH}}"
 	local runtime_repo="github.com/kata-containers/runtime"
 	GOPATH=${GOPATH:-${HOME}/go}
-	# We will not query the local versions.yaml file here to allow releases to
-	# always get the version from the committed tree. For our CI,
-	# .ci/install_kata_kernel.sh file in tests repository will pass the kernel
-	# version as an override to this function to allow testing of kernels before
-	# they land in tree.
-	versions_file="versions-${branch}.yaml"
+	# For our CI, we will query the local versions.yaml file both for kernel and
+	# all other subsystems. eg: a new version of NEMU would be good to test
+	# through CI. For the kernel, .ci/install_kata_kernel.sh file in tests
+	# repository will pass the kernel version as an override to this function to
+	# allow testing of kernels before they land in tree.
+	if ${CI}; then
+		versions_file="${GOPATH}/src/${runtime_repo}/versions.yaml"
+	else
+		versions_file="versions-${branch}.yaml"
```

Comment:
Since there are now multiple possible versions files, it might be useful to add an `info` or `echo` call so that it's clear from the test logs which one was used.